### PR TITLE
Some changes in bash scripts

### PR DESCRIPTION
**Pull request by luxiaohan**
1. Replace all the 'awk' commands with 'gawk'

2. Change some functions in dot_proflie and set_env.sh  to make syenv optional

3. Replace all the tabs with white-spaces

### CODE REVIEW

**Review by amereghe — CHANGES_REQUESTED** [review]
Hi Xiaohan,
many thanks for the work!

Some comments:
   * `awk` --> `gawk`: I saw that you also update the path to awk scripts: good move, thanks. Have you checked that no other `awk` statement was left behind?
   *  tabs: have you checked that no other tab char was left behind?

Have you run some tests? I guess you did, but just to be sure.

I had a further request by Frederik: could you add a `aklog` command anywhere you find `kinit`? The command should be placed right after and look like: `aklog -c cern.ch -k CERN.CH`
Following this change, the manual should be changed also, in Tab.~1.2, where we state the pre-requisites.

Thanks!

**Comment by luxiaohan** [comment]
Hello Alessio,
     Yes, I checked with the command ```grep```. So I'm sure there aren't ```awk``` statement and tab char left behind, but just in the ```bash``` folder.
      Of course, I did some tests, submitted madx jobs  and sixtrack jobs to HTCondor, I didn't see any error. But I think you'd better do some tests as well for double check.
       And ok, I will add the statement for the new request asap, and I will update this pull request after the changes are done.
Thanks~

**Comment by amereghe** [comment]
Hello Xiaohan,
ok, no worries, if you have run the tests, then it is fine with me.
Thanks for taking into account also the new request.

File: utilities/bash/control.sh [hunk 1]
```diff
@@ -297,9 +297,11 @@ if ${lkinit} ; then
     if ${lkrenew} ; then
         echo " --> kinit -R beforehand:"
         kinit -R
+        aklog -c cern.ch -k CERN.CH
     else
         echo " --> kinit beforehand:"
         kinit
+        aklog -c cern.ch -k CERN.CH
```
Comment (amereghe) [inline utilities/bash/control.sh]: I would actually do something like:
   1. `kinit` (or `kinit -R`);
   1. check that `kinit` did not fail;
   1. `aklog`;
   1. check that `aklog` did not fail;
   1. `klist`

File: utilities/bash/dot_profile [hunk 2]
```diff
@@ -1613,7 +1613,7 @@ function sixdeskRenewKerberosToken(){
         local __lKlist=true
     fi
     sixdeskmess -1 " --> kinit;"
-    multipleTrials "kinit -R ; local __exit_status=\$?" "[ \$__exit_status -eq 0 ]"
+    multipleTrials "kinit -R ; local __exit_status=\$?; aklog -c cern.ch -k CERN.CH" "[ \$__exit_status -eq 0 ]"
```
Comment (amereghe) [inline utilities/bash/dot_profile]: similarly to `control.sh`, I would keep `kinit` embedded in the `multipleTrials` algorithm, with the check of its success, and then have the `aklog` command, again as argument to the `multipleTrials` algorithm and a check afterwards

File: utilities/doc/chIntroduction.tex [hunk 3]
```diff
@@ -66,7 +66,7 @@ \section{Assumed Environment}\label{Sec:Env}
     \rowcolor{blue!30}
     \textbf{Component} & \textbf{reason} \\
     \hline
-    kerberos & to renew/check credentials via \texttt{klist} and \texttt{kinit} \\
+    kerberos & to renew/check credentials via \texttt{klist}, \texttt{kinit} and \texttt{aklog} \\
```
Comment (amereghe) [inline utilities/doc/chIntroduction.tex]: I think `aklog` should be in the AFS line.
BTW: could you change:
`AFS (local mount)`
into:
`AFS (or openAFS for local mounting)`

Also:
`kerberos`
into:
`kerberos (local installation)`

**Review by amereghe — CHANGES_REQUESTED** [review]
Thanks a lot, Xiaohan, for the work. I think that these modifications still need a bit of reworking. Thanks!

**Comment by luxiaohan** [comment]
Hi Alessio, 
   I have made the changes according to your comments!
Thanks~

**Review by amereghe — APPROVED** [review]

**Pull request merged**